README.md

```diff
@@ -8,58 +8,72 @@ appropriate helm command based on which action it is included within: `install`,
 
 ### Install or Upgrade
 
-```
+```shell
 porter mixin install helm
 ```
 
+### Configure mixin repositories
+
+```yaml
+- helm:
+      repositories:
+          stable:
+              url: "https://kubernetes-charts.storage.googleapis.com"
+              cafile: "path/to/cafile"
+              certfile: "path/to/certfile"
+              keyfile: "path/to/keyfile"
+              username: "username"
+              password: "password"
+```
+
 ### Mixin Syntax
 
 Install
 
 ```yaml
 install:
-- helm:
-    description: "Description of the command"
-    name: RELEASE_NAME
-    chart: STABLE_CHART_NAME
-    version: CHART_VERSION
-    namespace: NAMESPACE
-    replace: BOOL
-    devel: BOOL
-    wait: BOOL # default true
-    set:
-      VAR1: VALUE1
-      VAR2: VALUE2
+    - helm:
```

Comment:
I might request that the spacing adjustments here and below be discarded... and/or ensure the spacing is in increments of 2 rather than 4 for indents.